index.html.in

```diff
@@ -369,15 +369,15 @@
         <div id="footer">
             <div class="navbar-bottom hidden-xs">
                 <div class="container" id="footer-container">
-                    <ul class="nav nav-pills pull-left" style="display: inline-block;">
+                    <ul class="nav nav-pills pull-left d-inline-block ">
```

Comment:
@sushain97 ,this was the extra space you were refering to right?

Reply:
No.